gcore-spoofax-tests/ldbc-snb-interactive.spt

```diff
@@ -0,0 +1,24 @@
+module ldbc-snb-interactive
+
+language gcore
+
+
+test Query 1 [[
+
+SELECT friend.id, friend.lastName, length(p) AS distance, friend.birthday, friend.creationDate, friend.gender,
+       friend.browserUsed, friend.locationIP, friend.email, friend.speaks, friendCity.name,
+       ( SELECT GROUP_CONCAT(uni.name + ' ' + studyAt.year + ' ' + uniCity.name)
+         MATCH (friend) -[studyAt:study_at]-> (uni:University) -[:is_located_in]-> (uniCity:City) ON snbGraph100
+         /* how to GROUP BY friend while returning a single scalar from the SELECT? */
```

Comment:
this is actually an issue if we want to support scalar subqueries like in SQL, where it is required that the returned table has 1 row and 1 column